backend/src/main/java/de/bund/digitalservice/ris/search/utils/RisHighlightBuilder.java

```diff
@@ -66,6 +66,12 @@ private static HighlightBuilder addNormsFields(HighlightBuilder builder) {
   private static HighlightBuilder addCaseLawFields(HighlightBuilder builder) {
     CASE_LAW_HIGHLIGHT_CONTENT_FIELDS.forEach(builder::field);
     builder.field(getFieldBasic(CaseLawDocumentationUnit.Fields.ECLI));
+    /*
+     * Note that this uses the .text variant of file_numbers
+     * */
+    builder.field(
+        getFieldBasic(CaseLawDocumentationUnit.Fields.FILE_NUMBERS_TEXT)
+            .numOfFragments(5) /* Support highlighting multiple (partial) file numbers */);
```

Comment:
@hmones I've added a special case for highlighting file numbers since our pair programming session, just like we already did for ECLIs. The reason was that setting the default `numOfFragments` to something higher than 1 meant that multiple matches for e.g. headline would start to appear.

Reply:
Well done, I think that approach is good.